network/wsNetwork.go

```diff
@@ -90,6 +90,15 @@ const connectionActivityMonitorInterval = time.Minute * 3
 // we discard the connection.
 const maxPeerInactivityDuration = time.Minute * 5
 
+// maxMessageQueueDuration is the maximum amount of time a message is allowed to be waiting
+// in the various queues before being sent. Once that deadline has reached, sending the message
+// is pointless, as it's too stale to be of any value
+const maxMessageQueueDuration = time.Second * 25
```

Comment:
I think more idiomatic (and consistent with elsewhere in our code base, and more natural to read as "25 seconds") is `25 * time.Second`

Reply:
sounds like a good suggestion. I'll also update the above constants to align with that.